pull_request_template.md

```diff
@@ -0,0 +1,34 @@
+# Description
+Bridge route returns bridge data with community served data by bridge.
```

Comment:
Might be better on the template to change this to something along the lines of "Description of what this pull request includes."

Reply:
Very true, ty for pointing that out.